### ISSUE TITLE
Support for GameFAQs/Gamespot HTML guides 

### ISSUE THREAD

**Issue by Too-Ticki** [dteviot/WebToEpub#573]
Hello, GameFAQs/Gamespot has multi-chapter HTML guides that would be perfect to turn into epubs, except I can't get it to work, so if you could take a look I would really appreciate it. Here's what I've figured out so far, per the form example:

Table of Contents (list of chapters) of a typical story on the site:
https://gamefaqs.gamespot.com/switch/248065-luigis-mansion-3/faqs/78141

Did you try using the Default Parser for the site?  If not, why not?
Yes, but it turns out blank, see below

What settings did you use?
* URL of first chapter
https://gamefaqs.gamespot.com/switch/248065-luigis-mansion-3/faqs/78141/a-warm-suspicious-welcome
* CSS selector for element holding content to put into EPUB
.ffaqbody
* CSS selector for element holding Title of Chapter
.page-title
* CSS selector for element(s) to remove
.ftoc

What didn't work?
It shows up blank.
Using the CSS selectors above everything looks fine in the test window, but after clicking "Apply" and then "Pack epub", the generated epub is blank, and I get the following error message:
Error: Could not find content element for web page 'https://gamefaqs.gamespot.com/switch/248065-luigis-mansion-3/faqs/a-warm-suspicious-welcome'.

Even using the basic default body selector to capture everything the part of the pages where the text of the guide itself should be shows up blank.

If the Default Parser did not work, if you have developer skills, did you try writing a new parser?
I have no idea how to fix that the text shows up blank.

If you don't have developer skills, can you ask a friend who does have them if they can do it for you?
None of them know how to fix the blank text issue either.

Thank you for all your work!

**Comment by dteviot** [comment]
@Too-Ticki
Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source" instructions at https://github.com/dteviot/WebToEpub#how-to-install-from-source and let me know how it goes.

Painful due to different layouts of Table of Contents pages.
Works for 
* https://gamefaqs.gamespot.com/switch/248065-luigis-mansion-3/faqs/78141 and 
* https://gamefaqs.gamespot.com/pc/164594-fallout-4/faqs/72575

not so great for 
https://gamefaqs.gamespot.com/pc/164594-fallout-4/faqs

For my notes, 1 hour so far.

**Comment by Too-Ticki** [comment]
@dteviot 
Tried it, and as far as I can tell everything works without a hitch. Aside from the link in my first post, I also tried it with these two Guides:
https://gamefaqs.gamespot.com/ps3/998014-ni-no-kuni-wrath-of-the-white-witch/faqs/65904
https://gamefaqs.gamespot.com/wii-u/633263-pikmin-3/faqs/72517
and both seemed to render fine (although the second one took a while, probably because of its size.)

Hopefully the problematic layout you found is less common (maybe it's because that one was an official GameFAQs guide? I haven't stumbled on one of those yet.)

Thank you so much for sorting it all out!
